src/OSPSuite.Core/Domain/Builder/ExpressionProfileBuildingBlock.cs

```diff
@@ -0,0 +1,50 @@
+﻿using OSPSuite.Core.Domain.Services;
+
+namespace OSPSuite.Core.Domain.Builder
+{
+   public class ExpressionProfileBuildingBlock : StartValueBuildingBlock<ExpressionParameter>
+   {
+      public override string Icon => Type.IconName;
+
+      public virtual string MoleculeName { get; private set; }
+
+      public string Species { get; private set; }
+
+      public ExpressionType Type { set; get; }
+
+      public int PKSimVersion { set; get; }
+
+      public virtual string Category { get; private set; }
+
+      public override string Name
+      {
+         get => PKSimCoreConstants.ContainerName.ExpressionProfileName(MoleculeName, Species, Category);
```

Comment:
PKSimCoreConstants...not sure about this naming. Can we move this to the usual constant (and also use it in PKSim?)